README.md

```diff
@@ -56,6 +127,11 @@ There are `make` commands that alias some of this functionality:
 ### Running tests
 #### Sandbox Tests
 
+To start the sandbox, run the install command if not ran already
```

Comment:
Wording slightly confusing here, maybe remove 'To start the sandbox', as technically, we're running 'make install'